mvp-gate-party-grpc/src/main/proto/cz/tyckouni/mvpgate/party/Sp.proto

```diff
@@ -1,16 +1,33 @@
 syntax = "proto3";
 import "mvpgate/Base.proto";
+import "google/protobuf/empty.proto";
 
 package cz.tyckouni.mvpgate.party;
 
+
 option java_multiple_files = true;
 option java_package = "cz.tyckouni.mvpgate.party.grpc";
 
 service SpService {
   rpc findByGuid(mvpgate.ByGuidRequest) returns (SpGrpc) {}
+  rpc create(mvpgate.party.CreateRequest) returns (SpGrpc) {}
+  rpc findByName(mvpgate.party.ByNameRequest) returns (SpsGrpc) {}
+  rpc delete(mvpgate.ByGuidRequest) returns (google.protobuf.Empty) {}
 }
 
 message SpGrpc {
   string guid = 1;
   string name = 2;
+}
+
+message SpsGrpc {
+  repeated SpGrpc sps = 1;
+}
+
+message CreateRequest {
```

Comment:
We should probably name this something like `CreateSpRequest`.

Reply:
Changed.